src/components/login-view/login-view.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+
+export const LoginView = ({ onLoggedIn }) => {
```

Comment:
You should also add `propTypes` to the `login-view`, since you have the `onLoggedIn` `props`. It would be best to do the same for the `props` in `movie-view`.